src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ModifyRolesForUserTest.java

```diff
@@ -0,0 +1,169 @@
+        List<HashMap> professionalUsersResponses = (List<HashMap>) searchResponse.get("users");
+        HashMap professionalUsersResponse = professionalUsersResponses.get(1);
+
+        assertThat(professionalUsersResponse.get("idamStatus")).isNotNull();
+        assertThat(professionalUsersResponse.get("userIdentifier")).isNotNull();
+        String userId = (String) professionalUsersResponse.get("userIdentifier");
+
+        log.info("User Id::" + userId);
+        ModifyUserProfileData modifyUserProfileData = new ModifyUserProfileData();
+        RoleName role1 = new RoleName("pui-user-manager");
+        Set<RoleName> roles = new HashSet<>();
+        roles.add(role1);
+        modifyUserProfileData.setRolesAdd(roles);
+
+        Map<String, Object> modifiedUserResponse =  professionalApiClient.modifyUserRoleToExistingUserForPrdAdmin(HttpStatus.OK,modifyUserProfileData,orgIdentifier,userId);
+
+        Map<String, Object> searchResponse1 = professionalApiClient.searchAllActiveUsersByOrganisation(orgIdentifier, hmctsAdmin, HttpStatus.OK);
+        List<HashMap> professionalUsersResponses1 = (List<HashMap>) searchResponse1.get("users");
+        HashMap professionalUsersResponse1 = professionalUsersResponses1.get(1);
+        assertThat(professionalUsersResponse1.get("roles")).isNotNull();
+
+        ArrayList<String> rolesSize = ( ArrayList<String>)professionalUsersResponse1.get("roles");
+        assertThat(rolesSize.size()).isEqualTo(3);
+        assertThat(rolesSize.contains("caseworker,pui-organisation-manager,pui-user-manager"));
+    }
+
+    @Test
+    public void ac2_add_role_existing_user_using_pui_user_manager_for_external_200() {
+
+        Map<String, Object> searchResponse = professionalApiClient.searchAllActiveUsersByOrganisationExternal(HttpStatus.OK,generateBearerTokenForPuiManager(),"Active");
+        List<HashMap> professionalUsersResponses = (List<HashMap>) searchResponse.get("users");
+        HashMap professionalUsersResponse = professionalUsersResponses.get(0);
+        assertThat(professionalUsersResponse.get("idamStatus")).isNotNull();
+        assertThat(professionalUsersResponse.get("userIdentifier")).isNotNull();
+        String userId = (String) professionalUsersResponse.get("userIdentifier");
+
+        ModifyUserProfileData modifyUserProfileData = new ModifyUserProfileData();
+        RoleName role1 = new RoleName("pui-organisation-manager");
+        Set<RoleName> roles = new HashSet<>();
+        roles.add(role1);
+        modifyUserProfileData.setRolesAdd(roles);
+        Map<String, Object> modifiedUserResponse =  professionalApiClient.modifyUserRoleToExistingUserForExternal(HttpStatus.OK,modifyUserProfileData,bearerTokenForPuiUserManager,userId);
+        Map<String, Object> searchResponse1 = professionalApiClient.searchAllActiveUsersByOrganisation(orgIdentifierResponse, hmctsAdmin, HttpStatus.OK);
+        List<HashMap> professionalUsersResponses1 = (List<HashMap>) searchResponse1.get("users");

```

Comment:
Please change to reference the interface:
List<Map> professionalUsersResponses1 = (List<Map>) searchResponse1.get("users");

Reply:
implemented the review comment.